itou/www/siae_evaluations_views/views.py

```diff
@@ -663,18 +663,26 @@ def siae_submit_proofs(request, evaluated_siae_pk):
         evaluation_campaign__evaluations_asked_at__isnull=False,
         evaluation_campaign__ended_at=None,
     )
-    if evaluated_siae.state != evaluation_enums.EvaluatedSiaeState.SUBMITTABLE:
+    if not evaluated_siae.can_submit:
+        # This should not happen since the form to POST here only appear if can_submit is True
+        # but if someone manages to POST it, tell the user it didn't work
+        messages.error(request, "Impossible de soumettre les documents")
```

Comment:
```suggestion
        messages.error(request, "Impossible de soumettre les documents.")
```

Autrement, pour s’éviter le problème, il suffit de rajouter `submission_freezed_at__lt=timezone.now()` dans le `get_object_or_404`.

Reply:
Je préfère réutiliser le même `can_submit` que dans le template